libr/bin/p/bin_pe.c

```diff
@@ -108,10 +108,13 @@ static char *signature (RBinFile *bf, bool json) {
 	}
 	struct PE_ (r_bin_pe_obj_t) * bin = bf->o->bin_obj;
 	if (json) {
-		RJSVar *json = r_pkcs7_cms_json (bin->cms);
-		char *c = r_json_stringify (json, false);
-		r_json_var_free (json);
-		return c;
+		PJ *pj = r_pkcs7_cms_json (bin->cms);
+		char *json_str = strdup((char *)pj_string(pj));
```

Comment:
move `char *json_str = strdup((char *)pj_string(pj));` inside the `if`.
after that, i'll merge

Reply:
yep done